core/pom.xml

```diff
@@ -33,6 +33,12 @@
 
   <dependencies>
 
+    <dependency>
```

Comment:
You need to add this dependency in parent pom and use here as a reference, no need to verify version then. See how other dependencies are specified in this pom

Reply:
Okay this will be resolved!